src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
 
 export default class Fall_GM extends React.Component {
 	render() {
-		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT 
-	
-		const renderHumanReadableTime = ({ days, hours, minutes, seconds, completed }) => {
-			if (completed) return <h1 id="time-to-gm-headline-now">Fall General Meeting 2020 is happening right now!</h1>; //EVAN TODO: add the rest of the program and links to zoom / slack here
-	
-			let dayString ="Day";
-			let hourString = "Hour";
-			let minuteString = "Minute";
-			let secondString = "Second";
-	
-			if (days !== 1) dayString += 's';
-			if (hours !== 1) hourString += 's';
-			if (minutes !== 1) minuteString += 's';
-			if (seconds !== 1) secondString += 's';
+		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT
+
+		const renderCountdownOrNow = ({ days, hours, minutes, seconds, completed }) => {
+			if (completed) return <GM_NOW/>;
+			else {
+				let dayString = "Day";
+				let hourString = "Hour";
+				let minuteString = "Minute";
+				let secondString = "Second";
+				if(days !== 1) dayString += "s";
+				if(hours !== 1) hourString += "s";
+				if(minutes !== 1) minuteString += "s";
+				if(seconds !== 1) secondString += "s";
+
+				return <GM_COUNTDOWN days={days} hours={hours} minutes={minutes} seconds={seconds}
+				dayString={dayString} hourString={hourString} minuteString={minuteString} secondString={secondString}/>
```

Comment:
I'm not terribly happy with how the calculations are done here instead of within the `GM_COUNTDOWN` component itself, but to keep the diff relatively straightforward I opted for this pattern rather than moving the logic into the new component

Reply:
I think this is fine - `GM_COUNTDOWN` can just take the processed datetime and render the UI. I would prefer to make a helper function that encapsulates the logic you have here tho.